source/index.rst

```diff
@@ -1,6 +1,6 @@
-=====================================
```

Comment:
Super overline is necessary. More is better than less to avoid warnings when `|platform|` gets replaced with `Elastic Kubernetes Service`